articles/openshift/oshift-how-expand-persistent-vols.md

```diff
@@ -0,0 +1,162 @@
+logging-es-data-master-7bqadxxd   1          1         1
+logging-kibana                    1          1         1         config
+```
+>[!INFO] In this example, the `logging-kibana` pod depends on the `logging-es-data-master` pod so it is good practice to also scale this down during the expansion of the persistent volume for `logging-es-data-master`
+```
+$ oc scale dc logging-kibana --replicas=0
+deploymentconfig.apps.openshift.io/logging-kibana scaled
+$ oc scale dc logging-es-data-master-7bqadxxd --replicas=0
+deploymentconfig.apps.openshift.io/logging-es-data-master-7bqadxxd scaled
+```
+Confirm that the pod which uses the volume has been Terminated:
+```
+$ oc get pods | grep logging-es-data-master
+# (nothing returned)
+```
+
+## Resize the persistent volume
+
+Once the pod(s) dependent on the volume are stopped, we can edit the pvc to resize the volume.
+
+```
+$ oc get pvc
+NAME           STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+logging-es-0   Bound     pvc-4ad7258e-12a6-11e9-bc9d-fa163e12bba5   20Gi       RWO            tier2          2d
+$ oc edit pvc logging-es-0
+...
+```
+`oc edit` causes the configuration of the object to open in a text editor. Edit the size in the `spec:` section only (do not touch the size listed under `status:`):
+```
+...
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 30Gi   # (edited from 20Gi)
+  storageClassName: tier2
+...
+```
+When the edit is completed, the pvc will *continue to show the old size* but will be listed as being in a `FileSystemResizePending` state:
+```
+oc describe pvc logging-es-0
+Name:          logging-es-0
+Namespace:     openshift-logging
+StorageClass:  tier2
+Status:        Bound
+Volume:        pvc-4ad7258e-12a6-11e9-bc9d-fa163e12bba5
+Labels:        logging-infra=support
+Annotations:   pv.kubernetes.io/bind-completed=yes
+               pv.kubernetes.io/bound-by-controller=yes
+               volume.beta.kubernetes.io/storage-provisioner=kubernetes.io/cinder
+Finalizers:    [kubernetes.io/pvc-protection]
+Capacity:      20Gi
+Access Modes:  RWO
+Conditions:
+  Type                      Status  LastProbeTime                     LastTransitionTime                Reason  Message
+  ----                      ------  -----------------                 ------------------                ------  -------
+  FileSystemResizePending   True    Mon, 01 Jan 0001 00:00:00 +0000   Thu, 11 Apr 2019 17:43:26 +0000           Waiting for user to (re-)start a pod to finish file system resize of volume on node.
+Events:                     <none>
+```
+
+## Restart the pod(s)
+
+When the pod which uses the persistent volume is started, the persistent volume will be resized as it starts.
+
+Since we scaled down the `logging-es-data-master` deployment and its dependent `logging-kibana` in this example, we will scale up both deployments.
+```
```

Comment:
Same as previous comment (https://github.com/UKCloud/documentation/pull/500/files#r278072241) could we use:

```
$ for i in $(oc get dc | awk 'NR>1 {print $1}') ; do oc scale --replicas=1 dc $i; done
deploymentconfig.apps.openshift.io/logging-kibana scaled
deploymentconfig.apps.openshift.io/logging-es-data-master-7bqadxxd scaled
```